src/pyOpenMS/pxds/MzDataFile.pxd

```diff
@@ -13,13 +13,39 @@ cdef extern from "<OpenMS/FORMAT/MzDataFile.h>" namespace "OpenMS":
         # wrap-inherits:
         #   ProgressLogger
 
-        MzDataFile() nogil except +
-
-        void load(const String&, MSExperiment &) nogil except+
-        void store(const String&, MSExperiment &) nogil except+
-
-        PeakFileOptions getOptions() nogil except +
-        void setOptions(PeakFileOptions) nogil except +
+        MzDataFile() nogil except + # wrap-doc:File adapter for MzData files
+
+        void load(const String&, MSExperiment &) nogil except +
+            # wrap-doc:
+            #   Loads a map from a MzData file
+            #   -----
+            #   :param map: It has to be a MSExperiment or have the same interface
```

Comment:
add filename

Reply:
Sorry, I didn't understand. Can you please elaborate?

Reply:
The comment was about the param filename - which you added already :)